packages/models/src/Callback.ts

```diff
@@ -13,4 +13,13 @@ export interface Callback {
   isLogicFunctionProvided: boolean
   renderArguments: string[]
   isRenderFunctionProvided: boolean
+  results: CallbackResult[]
```

Comment:
I have a strong objection to the identifier `results`.  I don't think this at all communicates the idea that the results are static / mock values for use in generating deterministic dialog replay.

A few alternative ideas :

- mockResults
- stubResults
- resultsForReplay
- staticReplayResults

Reply:
It's not only for replay so bottom two are misleading. User might not even know what replay is and doesn't need to be concerned about that. They might think these are only for replay and disregard them.  It's more important for creating dialogs as we want users to define these up front and use them during training.

In my opinions they are closest to a list of mock logic result objects. Logic result is already used internally the SDK and has concept of entities and return value. But this name is bit weird for users to see.

I think mocks are static objects and stubs are functions.  callbacks provide `logic` and `render` functions and when it was called `stubs` it was a list of stub callbacks. When stub was chosen we call those functions instead of the actual functions.  This is very close to how stubs work everywhere and expected to feel familiar.

Because we're not doing functions anymore so feels wrong to keep calling them stubs. There more like mocks, because they're static objects but it's still feels weird to call them mocks because they're not mocking any real object the user can observe or construct in real callbacks. Thus there is no object to mimic or mock. Real callbacks use memory manager to set and effect things they don't return objects.

Also, it used to represent the whole callback. Now that it's only simulating the logic function.

Will let you and Lars decide.  Already spent too much time on renaming from stub to callback result. Whatever the next name is please pick one and make it final.

Reply:
Just caught up with Lars.  Let's go with `mockResults`.  Thanks!

Reply:
Ok, renamed.